app/main/routes.py

```diff
@@ -358,11 +376,17 @@ def browse_consignment(_id: uuid.UUID):
     from_date = None
     to_date = None
     date_filters = {}
+    date_error_fields = []
 
     if len(request.args) > 0:
-        date_validation_errors, from_date, to_date, date_filters = (
-            validate_date_filters(request.args, browse_consignment=True)
-        )
+        if len([k for k, v in request.args.items() if "date" in k]):
```

Comment:
whats this new condition for?

Reply:
When the user already received a validation error on the consignment view and when they click on clear filters, the validation error remains , to avoid that a check has been made to see if there is no 'date' filter values, it should not perform date validations

Reply:
as shown the demo on huddle, the code is required to work for clear filters